lib/apiservers/engine/backends/image.go

```diff
@@ -184,6 +155,7 @@ func (i *Image) PullImage(ctx context.Context, ref reference.Named, metaHeaders
 		return err
 	}
 
+	layerCache.Update()
```

Comment:
Ok, since it doesn't show up on my view either, I will add the comment here.  You're doing a full update of the cache on every pull.  That seems rather excessive.  Can't we just get the metadata for that one image and save it to the cache?